scripts/devel/create_devel_task_generator.py

```diff
@@ -124,6 +124,8 @@ def main(argv=sys.argv[1:]):
         ]
     elif 'catkin' not in pkg_names:
         debian_pkg_names += resolve_names(['catkin'], **context)
+    if args.ros_version == 2:
+        debian_pkg_names.append('ros-%s-ros-workspace' % (args.rosdistro_name,))
```

Comment:
Is there a reason not to use the same logic as catkin does above?
```suggestion
        debian_pkg_names.append(resolve_names(['ros_workspace'], **context))
```

Similarly a reason to use `.append()` over `+=`?

Reply:
ea55278